projects/ontimize-web-ngx/src/lib/layouts/form-layout/split-pane/o-form-layout-split-pane.component.html

```diff
@@ -0,0 +1,21 @@
+<div fxFill>
```

Comment:
Is this `div` needed? Can we move the `fxFill` to the next `div` that already has `height: 100%` in the class `row-wrapper`